src/components/ComparisonView.tsx

```
@@ -0,0 +1,86 @@
+  removeFromComparison,
+  clearComparison,
+  setComparisonMode
+}: ComparisonViewProps) {
+  return (
+    <div className="bg-white rounded-xl shadow-lg p-6 border border-gray-200 mb-6">
+      <div className="flex items-center justify-between mb-4">
+        <h2 className="text-xl font-semibold text-gray-900">Compare Vehicles</h2>
+        <button
+          onClick={() => setComparisonMode(false)}
+          className="text-gray-500 hover:text-gray-700 transition-colors"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)